cmd/hostd/main.go

```diff
@@ -104,20 +102,8 @@ func tryLoadConfig() {
 		configPath = str
 	}
 
-	// If the config file doesn't exist, don't try to load it.
-	if _, err := os.Stat(configPath); os.IsNotExist(err) {
-		return
-	}
-
-	f, err := os.Open(configPath)
-	checkFatalError("failed to open config file", err)
-	defer f.Close()
-
-	dec := yaml.NewDecoder(f)
-	dec.KnownFields(true)
-
-	if err := dec.Decode(&cfg); err != nil {
-		fmt.Println("failed to decode config file:", err)
+	if err := config.LoadFile(configPath, &cfg); err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
`nit` since we don't really use `os.ErrNotExists` we might as well have `config.LoadFile` obfuscate it so the caller doesn't need to worry about it

Reply:
I want to leave it up to the caller what to do instead of turning load into an effective `TryLoad`